src/tests/Common/CLRTest.Jit.targets

```diff
@@ -222,7 +222,10 @@ IF NOT DEFINED DoLink (
           DependsOnTargets="GetDisasmCheckData">
     <PropertyGroup>
       <HasBashDisasmCheck>false</HasBashDisasmCheck>
-      <HasBashDisasmCheck Condition="'$(HasDisasmCheck)' == 'true' and '$(RuntimeFlavor)' == 'coreclr' and ('$(TargetOS)' == 'Linux' or '$(TargetOS)' == 'OSX') and ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'arm64')">true</HasBashDisasmCheck>
+      <HasBashDisasmCheck Condition="'$(HasDisasmCheck)' == 'true' and '$(RuntimeFlavor)' == 'coreclr' and ('$(TargetOS)' == 'Linux' or '$(TargetOS)' == 'OSX') and ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'arm64') and '$(RunCrossGen2)' != 'true'">true</HasBashDisasmCheck>
```

Comment:
I think you could fold the RunCrossGen2 check into GetDisasmCheckData and then not duplicate it on the bash/batch sides.

Reply:
Makes sense, I can also do the setting of `GCStressIncompatible/HeapVerifyIncompatible` well too.

Reply:
I think there's a problem with the two Incompatible vars.  Since it's a separate target, those definitions might be "local" to it.   Put another way, those properties only act like "global variables" when msbuild files are being `Import`ed.

To do this, you'd need to update the `Returns` list.  The name `GetDisasmCheckData` would then be a bit misleading - it's now setting things outside of disasm based on the disasm setting.  I just ran an experiment, and if the "subtarget" doesn't actually define the property, then the "caller target"'s value is left alone.